providers/providertest/util.go

```diff
@@ -15,6 +15,9 @@ func FilterMarketMapToProviders(mm mmtypes.MarketMap) map[string]mmtypes.MarketM
 	for _, market := range mm.Markets {
 		// check each provider config
 		for _, pc := range market.ProviderConfigs {
+			// remove normalizations to isolate markets
+			pc.NormalizeByPair = nil
```

Comment:
Is this just for testing, or why is it needed?

Reply:
This is so that we can actually test the markets in isolation.

For example, if I want to isolate all of the Uniswap markets, they're all going to be Normalized by ETH/USD, which requires a bunch of other feeds to exist.

For this kind of testing, we only want to see "Does uniswap return a price for this expected pool?", so we just remove normalize pairs